src/main/java/io/jmix/uisamples/view/flowui/components/fileupload/filestorageuploadfield/FileStorageUploadFieldSample.java

```diff
@@ -0,0 +1,46 @@
+package io.jmix.uisamples.view.flowui.components.fileupload.filestorageuploadfield;
+
+import com.vaadin.flow.component.notification.NotificationVariant;
+import com.vaadin.flow.component.upload.Receiver;
+import io.jmix.core.FileRef;
+import io.jmix.flowui.Notifications;
+import io.jmix.flowui.component.upload.FileStorageUploadField;
+import io.jmix.flowui.component.upload.receiver.FileTemporaryStorageBuffer;
+import io.jmix.flowui.kit.component.upload.event.FileUploadSucceededEvent;
+import io.jmix.flowui.upload.TemporaryStorage;
+import io.jmix.flowui.view.*;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.File;
+import java.util.UUID;
+
+@ViewController("file-storage-upload-field")
+@ViewDescriptor("file-storage-upload-field.xml")
+public class FileStorageUploadFieldSample extends StandardView {
+    @Autowired
+    private TemporaryStorage temporaryStorage;
+    @ViewComponent
+    private FileStorageUploadField fileStorageUploadField;
+    @Autowired
+    private Notifications notifications;
+
+    @Subscribe("fileStorageUploadField")
+    public void onFileStorageUploadFieldFileUploadSucceeded(
+            final FileUploadSucceededEvent<FileStorageUploadField> event) {
```

Comment:
Do we have a linter for exceeded line length? If not, i think it's better to not split method declaration into two lines